### PR TITLE
Fixes VSTS Bug 715151: [Feedback] 64 bit unit testing is not supported

### PR DESCRIPTION
**Pull request by mkrueger**
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/715151

Unit testing now values the platfom target setting.

### CODE REVIEW

**Review by Therzok — APPROVED** [review]

File: main/src/core/MonoDevelop.Core/MonoDevelop.Core.csproj [hunk 1]
```diff
@@ -780,6 +780,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="MonoDevelop.Core.Tests" />
     <InternalsVisibleTo Include="MonoDevelop.Ide.Tests" />
+    <InternalsVisibleTo Include="MonoDevelop.UnitTesting.NUnit" />
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Core/MonoDevelop.Core.csproj]: Don't like IVT here. Can't we expose this API somehow?

**Review by Therzok — CHANGES_REQUESTED** [review]
Remove IVT

File: main/src/core/MonoDevelop.Core/MonoDevelop.Core.csproj (hunk 1 above)
Comment (mkrueger) [reply main/src/core/MonoDevelop.Core/MonoDevelop.Core.csproj]: No I don't want to make that API public.

**Comment by mkrueger** [comment]
@Therzok: I don't want to change the public API. We should stop just popping in public helper methods from everywhere just because we need to use them internally.

**Review by Therzok — APPROVED** [review]

**Review by slluis — APPROVED** [review]
Nice!

**Pull request merged**